docker/bin/create_vanilla_dump

```diff
@@ -65,4 +65,4 @@ docker_create_db_directories
 
 log "running run_dump.sh using init_script=$init_script and dumpfile=$dump_file"
 
-exec gosu postgres /usr/local/bin/run_dump.sh $dump_file $init_script
\ No newline at end of file
+exec /usr/local/bin/run_dump.sh $dump_file $init_script
```

Comment:
newer postgres versions don't really like `gosu` (the given script is broken with it), but it works without it flawlessly